src/component-library/Balance/Balance.style.tsx

```diff
@@ -1,18 +1,19 @@
 import styled from 'styled-components';
 
 import { theme } from 'component-library';
+import { spaceX } from 'component-library/css';
 
 const BalanceWrapper = styled.dl`
   display: flex;
   font-weight: ${theme.fontWeight.book};
+  ${spaceX('spacing1')}
```

Comment:
@peterslany 
FYI: I added this `spaceX` utility helper function. I'd say it could be DX friendly. So I updated like so.

Reply:
Hmm, I'm not sure about the usage of this helper function. It makes the code harder to read for me because it hides the css property it sets, so I have to look into definition of the function to see what is going on. Also, what is an advantage of using this util over the `gap: ${theme.spacing.spacing1}`? Let me know your thoughts on this please :) @crypto-engineer @tomjeatt

Reply:
@peterslany yes I agree on this one - I would use suggest using `gap` here.

Reply:
Hmm, it's a little different use case from `gap` since gap can only be used with flex or grid. In the past, gap for flex was not supported in all browsers.
I have no strong opinion. If you like `gap` then I will use `gap` in this case.

Reply:
Thanks - it makes sense here because we're using `flex` (and we don't support browsers without flex support).